.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
 
 before_script:
 # Download kubectl, which is a requirement for using minikube.
-- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.7.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
This is intended to be pinned to that specific version as we also want to pin against the same version of the Kubernetes cluster we are starting a few lines below. Otherwise you risk a mismatch of API versions.

Reply:
But if you willing to, you can bump up versions to `v1.8.0`. 🙂